### ISSUE TITLE
Feature request: allow disabling the screen rotation

### ISSUE THREAD

**Issue by twisteros** [BelledonneCommunications/linphone-android#1605]

**Comment by twisteros** [comment]
Understand. I thought about that as my phone's gyroscope gets crazy when I'm in bed,
call a bank and need to enter digits in its voice menu.
Btw the stock caller doesn't rotate while in call.

**Comment by Viish** [comment]
Yes but while in video call it's a good thing to be able to rotate the phone to display video in landscape.

**Comment by narcisgarcia** [comment]
Some apps are welcome to rotate, and some users we prefer phone app to be static. In this situation, OS autorotate behaviour is enabled, and for concrete apps user disables autorotation.
Example for Linphone: To only hang up a call without button being displaced.
